docs/getting_started/requirements.md

```diff
@@ -56,7 +56,7 @@ For security and performance we generally recommend (unless otherwise noted) usi
         - Redis 4.0+ (5.0 recommended, using `volatile-*` [eviction policy](https://redis.io/topics/lru-cache) is required with default [Redis adapter](../guide/persistence_cache.md#redis))
         - Memcached 1.5 or higher (See [Memcached adapter](../guide/persistence_cache.md##memcached) for comparison with Redis)
     - Session: either own Redis instance with persistence turned on, or Database.
-    - Search: Solr 7 or Elasticsearch 7.7 (recommended over SQL-based Search engine, especially on cluster, as SQL does not provide the same feature set or performance as the other two).
+    - Search: Solr 7/8 or Elasticsearch 7.7 (recommended over SQL-based Search engine, especially on cluster, as SQL does not provide the same feature set or performance as the other two).
```

Comment:
```suggestion
    - Search: Solr 7, Solr 8, or Elasticsearch 7.7 (recommended over SQL-based Search engine, especially on cluster, as SQL does not provide the same feature set or performance as the other two).
```
Has Solr 8 been tested with Ibexa Platform?

Reply:
Unsure, it's part of the integration tests for some times but for manual tests you'll need to ask @ezsystems/qa-team